### PR TITLE
add digitalocean metadata

### PR DESCRIPTION
**Pull request by mitayai**

### CODE REVIEW

**Comment by chef-supermarket** [comment]
Hi. I am an automated pull request bot named Curry. There are commits in this pull request whose authors are not yet authorized to contribute to Chef Software, Inc. projects or are using a non-GitHub verified email address. To become authorized to contribute, you will need to sign the Contributor License Agreement (CLA) as an individual or on behalf of your company. [You can read more on Chef's blog.](https://www.chef.io/blog/2014/06/23/changes-to-the-contributor-license-agreement-process)

## GitHub Users Who Are Not Authorized To Contribute

The following GitHub users do not appear to have signed a CLA:

* @mitayai

[Please sign the CLA here.](https://supermarket.chef.io/become-a-contributor)

**Comment by chef-supermarket** [comment]
Hi. Your friendly Curry bot here. Just letting you know that all commit authors have become authorized to contribute. I have added the "Signed CLA" label to this issue so it can easily be found in the future.

**Comment by thommay** [comment]
@mitayai do you need help with those tests?

File: lib/ohai/plugins/digital_ocean.rb [hunk 1]
```diff
@@ -15,6 +15,7 @@
 # limitations under the License.
 
 require 'ohai/util/ip_helper'
+require 'json'
```
Comment (lamont-granquist) [inline lib/ohai/plugins/digital_ocean.rb]: should use ffi-yajl, not json gem

**Comment by mitayai** [comment]
The person who was helping me with this is no longer able to help me, and i'm not competent enough in ruby to complete this. Can someone else take it on and address any lingering concerns?

**Comment by GenPage** [comment]
After talking with @mitayai, i'll be able to take this over.

**Comment by tas50** [comment]
Once I wrap up the Openstack metadata work I'll take a look at making a few changes here and getting specs passing.  I'd love to get this merged in ASAP.

**Comment by tas50** [comment]
@mitayai We have a larger refactor of the DO plugin in #823 that will be merged in April for release with Chef 13. This removes a large portion of the old DO logic and relies entirely on the API. Take a look and please comment if that would help you out.

**Pull request closed without merging**